docs/getting_started/features/metadata.md

```diff
@@ -6,15 +6,47 @@ GTFS feeds can contain further information regarding the feed itself and its pro
 
 Feed Information communicates important information about the feed, such as its validity (start day and end date), the publishing organization, and contact information for inquiries regarding the GTFS dataset and data publishing practices.  
 
+**Pre-requirements**: Implement Base features.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[feed_info.txt](/schedule/reference/#feed_infotxt)|`feed_publisher_name`, `feed_publisher_url`, `feed_lang`, `default_lang`, `feed_start_date`, `feed_end_date`, `feed_version`, `feed_contact_email`, `feed_contact_url` |
+|[feed_info.txt](/documentation/schedule/reference/#feed_infotxt)|`feed_publisher_name`, `feed_publisher_url`, `feed_lang`, `default_lang`, `feed_start_date`, `feed_end_date`, `feed_version`, `feed_contact_email`, `feed_contact_url` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows additional information relative to the GTFS dataset, including its validity, version and contact information.
```

Comment:
This is repetitive of the description above. I would make it specific to the sample, or remove the description altogether since the sample is pretty self-explanatory.